packages/source-iottwinmaker/src/data-binding-provider/EntityPropertyBindingProviderStore.ts

```diff
@@ -220,10 +220,16 @@ export class EntityPropertyBindingProviderStore implements IValueDataBindingStor
       if (componentName) (dataBindingContext as ITwinMakerEntityDataBindingContext).componentName = componentName;
       if (propertyName) (dataBindingContext as ITwinMakerEntityDataBindingContext).propertyName = propertyName;
 
-      const dataBinding = {
+      const dataBinding: IValueDataBinding = {
         dataBindingContext,
       };
 
+      if (componentName && propertyName && this.selectedEntity) {
```

Comment:
Can `componentName` or `propertyName` be an empty string?

Reply:
they are getting from the entity so shouldn't be empty